storio-common-annotations-processor/src/main/java/com/pushtorefresh/storio/common/annotations/processor/StorIOProcessor.java

```diff
@@ -0,0 +1,235 @@
+package com.pushtorefresh.storio.common.annotations.processor;
+
+import com.pushtorefresh.storio.common.annotations.processor.generate.ResolverGenerator;
+import com.pushtorefresh.storio.common.annotations.processor.introspection.StorIOColumnMeta;
+import com.pushtorefresh.storio.common.annotations.processor.introspection.StorIOTypeMeta;
+
+import org.jetbrains.annotations.NotNull;
+
+import java.lang.annotation.Annotation;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+import javax.annotation.processing.AbstractProcessor;
+import javax.annotation.processing.Filer;
+import javax.annotation.processing.Messager;
+import javax.annotation.processing.ProcessingEnvironment;
+import javax.annotation.processing.RoundEnvironment;
+import javax.lang.model.SourceVersion;
+import javax.lang.model.element.Element;
+import javax.lang.model.element.TypeElement;
+import javax.lang.model.util.Elements;
+
+import static javax.lang.model.element.ElementKind.CLASS;
+import static javax.lang.model.element.Modifier.FINAL;
+import static javax.lang.model.element.Modifier.PRIVATE;
+import static javax.tools.Diagnostic.Kind.ERROR;
+
+/**
+ * Base annotation processor for StorIO
+ * <p>
+ * It'll process annotations to generate StorIO Object-Mapping
+ * <p>
+ * Addition: Annotation Processor should work fast and be optimized because it's part of compilation
+ * We don't want to annoy developers, who use StorIO
+ */
+// Generate file with annotation processor declaration via another Annotation Processor!
+public abstract class StorIOProcessor
```

Comment:
Let's rename it to `StorIOAnnotationsProcessor`? Just to make it more clear for the potential readers of the code!